tests/tck/features/bugfix/SubgraphBeforePipe.feature

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+Feature: Test get subgraph before pipe
+
```

Comment:
Add a case to access one column of the subgraph output.

Reply:
Cannot get the column of the output now since `._vertices` is a reserved keyword.

```
(root@nebula) [nba]> $a = GET SUBGRAPH WITH PROP 3 STEPS FROM 'Paul George' OUT serve BOTH like | yield $a._vertices[0]
[ERROR (-1004)]: SyntaxError: syntax error near `_vertice'
```

Reply:
ok. You could add the error case to tck.

Reply:
Got it.